lib/pbench/test/unit/server/test_cache_manager.py

```diff
@@ -68,12 +68,12 @@ def selinux_enabled(monkeypatch):
     monkeypatch.setattr("pbench.common.selinux.restorecon", lambda a: None)
 
 
-class TestFileTree:
+class TestCacheManager:
     def test_create(self, server_config, make_logger):
         """
-        Create an empty FileTree object and check the properties
+        Create an empty CacheManager object and check the properties
         """
-        tree = FileTree(server_config, make_logger)
+        tree = CacheManager(server_config, make_logger)
```

Comment:
It's interesting that you chose to rename all the local `FileTree` instances to `cache_m` in the product code but left `tree` in unit tests.

I'm not thrilled about `cache_m` (every time I see it, it seems "unfinished"), and we don't need to have total consistency between product code and unit tests, but it doesn't hurt either, and `tree = CacheManager(...)` doesn't really make much sense.

I would go with `cache` (which is admittedly overly generic) or maybe `cacheman` (which is more specific but/and also a bit silly)... but at least I'd like to advocate (lightly) for consistency and at least a better name on the testing side.

Reply:
For the most part, the references to `cache_m` are localized enough that we probably don't care much what they are named.

However, I'm with Dave that sticking with `tree` here seems a bit...anachronistic?  🙂

`cache` is OK with me.  (While `cacheman` would be OK, too, but, for a scope of just half a screen, even `cm` would be fine....)